### DISCUSSION TITLE
PPP Tests

### DISCUSSION THREAD

**Discussion by PaulWAyers** [theochem/ModelHamiltonian#29] [General]
Here are some references with the PPP Hamiltonian and data for testing. The second reference is especially helpful in terms of writing the Hamiltonian; the first in terms of  data.
https://aip.scitation.org/doi/abs/10.1063/1.476379
[s1380-7323%2899%2980019-3.pdf](https://github.com/theochem/ModelHamiltonian/files/8028370/s1380-7323.2899.2980019-3.pdf)

**Comment by gabrielasd** [comment]
http://vergil.chemistry.gatech.edu/notes/uga_hamil/uga_hamil.html shows how to get from QC Hamiltonian to the Hamiltonian in terms of the unitary group operators

Equivalent definitions of PPP Hamiltonian in the following references
Eq 2 and 4 in the 2nd reference from Paul: s1380-7323.2899.2980019-3.pdf
Also Eq 1 here https://doi.org/10.1002/qua.560510104
and Eq 16 in the reference bellow:
[2002 Jacek Karwowski - Relations between Pariser Parr Pople and Heisenber [retrieved 2022-03-18].pdf](https://github.com/theochem/ModelHamiltonian/files/8345056/2002.Jacek.Karwowski.-.Relations.between.Pariser.Parr.Pople.and.Heisenber.retrieved.2022-03-18.pdf)

**Reply by PaulWAyers** [reply]
Looking at the literature, the "R" value in in the PPP parameters is measured in Bohr. Specifically, that's true for the Mataga-Nishimoto formula
$$
\gamma_{ab} = \frac{1}{R_{ab} + \left(\tfrac{gamma_{aa}+\gamma_{bb}}{2}\right)^{-1}} 
$$
and the Ohno formula
$$
\gamma_{ab} = \frac{1}{\sqrt{R_{ab}^2+ \left(\tfrac{gamma_{aa}+\gamma_{bb}}{2}\right)^{-2}}} 
$$
This assumes that the $\gamma_{ab}$ terms are also in atomic units of Hartree.

**Reply by PaulWAyers** [reply]
Looking at some references, the "extra" one-body term that contributes may be
$$
\tilde{\alpha}_a = \alpha_a + \tfrac{1}{2} \gamma_{aa} = \alpha_a + \tfrac{U}{2}
$$

**Reply by PaulWAyers** [reply]
Referencing [J. Číẑek; F. Vinette (1988). Lower bounds for the ground state energy for the PPP and Hubbard models of the benzene molecule. , 149(5-6), 516–520. doi:10.1016/0009-2614(88)80374-9](https://www.sciencedirect.com/science/article/abs/pii/0009261488803749?via%3Dihub) it seems that for benzene we have:
$$
\gamma_{00} = 10.84 \text{ eV} \\
\gamma_{01} = \gamma_{10} = 5.27760 \text{ eV}  \\
\gamma_{02} = \gamma_{20} = 3.86206 \text{ eV} \\
\gamma_{03} = \gamma_{30} = 3.48785 \text{ eV}  \\
$$
More generally, this should be written as:
$$
\gamma_{a,a} = 10.84
\gamma_{a,a-1} = \gamma_{a,a+1} = 5.27760
\gamma_{a,a-2} = \gamma_{a,a+2} = 3.86206
\gamma_{a,a-3} = \gamma_{a,a+3} = 3.48786
$$
where additions/subtractions are considered modulo 5. 

For the Hubbard model Cizek and Vinnette use $U=5 \text{ eV}$. They mention that this should be about $\gamma_{00} - \gamma_{00}$. But I think that in the PPP model, the $U$ term is $\gamma_{00}/2$.

They report correlation energies (sadly, not total energies) for $\beta = 0.0,0.5,1,1.5,\ldots,5.0$.

Since Cizek and Paldus are very closely linked, it is reasonable to assume that Paldus uses the same values for $\gamma_{ab}$ in his papers. A somewhat more comprehensive set of correlation energies is avaliable [J. Paldus, M. Takahashi, R. W. H. Cho
Physical Review B, 30, 4267 (1984)](https://journals.aps.org/prb/abstract/10.1103/PhysRevB.30.4267). 

[J. Chem. Phys. 116, 480 (2002)](https://doi.org/10.1063/1.1425408) makes it very clear that only off-site (i.e., $\gamma_{a \ne b}$ should be included. The diagonal $\gamma_{aa}$ term doesn't include the "nuclear-nuclear" terms, unsurprisingly.

The most useful data (including HF and FCI energies) seems to be from Gagliardi, [Bendazzoli, G.L., Evangelisti, S. and Gagliardi, L. (1994), Full configuration interaction study of the ground state of closed-shell cyclic PPP polyenes. Int. J. Quantum Chem., 51: 13-25.](https://doi.org/10.1002/qua.560510104). Table 1 seems very useful, and note that E_{exact} = 0 for $beta=0$ even though $\gamma \ne 0$. 

They also give explicit formulas for the integrals (in physics notation it seems!) for any polyene (or at least any even polyene):
$$
\langle k | h | \rangle = h_{kl} = 2 \beta \delta_{kl} \cos\left(\frac{2 \pi k}{N} \right)
$$
$$
\langle< k m | l j \rangle = K(k-l) \delta{k+m,l+j}
$$
Here N is the number of sites (e.g., N=6) and 
$$
K(p) = \frac{1}{N} \sum{\mu=0}{N-1} \exp{\frac{2 p i \mu \pi}{N}}
$$
where it is understood that all indices are considered modulo N.  This is not going to look very much like the form we have, because they use a plane-wave basis instead of a site basis.

Directly interpreting their form, however, one expects to have the zero-body term:
$$
h_0 =\tfrac{1}{2} \sum_{\mu=0}^{N-1}\sum_{\nu=0}^{N-1} \gamma_{\mu \nu}
$$
the one-body terms:
$$
h_{\mu,\mu} = - \sum_{\nu=0}{N-1} \gamma_{\mu,\nu}
$$
$$
h_{\mu,\mu+1} = h_{\mu,\mu-1} =  \beta
$$
(All other 1-body terms are zero)
I think the 2-body terms are then $\gamma_{\mu,\nu}$ but it seems not unlikely that the diagonal terms should be halved, i.e., $$
\langle \mu \mu | \mu \mu \rangle = \tfrac{1}{2?} \gamma_{\mu,\mu} \\
\langle \mu \nu | \mu \nu \rangle = \gamma{\mu,\nu}
$$

**Comment by msricher** [comment]
Yep, those Vergil notes are what we need to do. It's in physicist notation
already, so we just need to work out what each of the coefficients in front
are equal to in terms of the polyene parameters.

**Reply by PaulWAyers** [reply]
What we really need to do is obtain all the "interconversion formulas" between different integral formats. We have a grasp(?) on the PPP model in terms of unitary-group generators (the "easy" way to write it). So this set of notes (writing "normal" integrals in UGA form) is the opposite of what we need.

**Reply by PaulWAyers** [reply]
[Karwowski and Flock](https://sci-hub.mksa.top/10.1016/s1380-7323(02)80020-6) give the following conversion to spatial integrals (chemists' notation)
$$
\left( p q | r s \right) = \gamma_{pr} \delta{pq} \delta{rs} - \gamma{pp} \delta{pr} \delta{rs} \delta{pr} 
$$
The strange bit here presumably arises because we sum over $p,q,r,s$ so we count the $p=r$ terms twice (positive) in the first term then we subtract them once (negative) in the second term. Maybe? I didn't do the math.

**Reply by PaulWAyers** [reply]
Looking at Sherrill's notes it seems increasingly likely that $U_{\mu \mu} = \tfrac{1}{2} \gamma_{\mu \mu}$

**Reply by PaulWAyers** [reply]
The Sherrill notes also appear (with internally defined notation) below.

http://vergil.chemistry.gatech.edu/notes/ci.pdf

It seems like the spatial orbital representation is pretty clear here; I wish we'd had that earlier in the project!
